### PR TITLE
Feature/tidy ups

### PR DESCRIPTION
**Pull request by noway**
Removed unused files and added some documentation in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Tests
+```
+npm test
+```
+
+## Usage
+
+```circom
+include "sha512/sha512/sha512.circom";
+include "circomlib/binsum.circom"; // make sure to include binsum
```
Comment (rahulghangas) [inline README.md]: Maybe change this to indicate that circomlib exists in node_modules

**Review by rahulghangas — CHANGES_REQUESTED** [review]
LGTM, just one tiny nitpick

**Comment by rahulghangas** [comment]
@noway and once again, thank you for your contribution!

File: README.md (hunk 1 above)
Comment (noway) [reply README.md]: Changed the comment. I wanted to signify that you can use your own copy of binsum from your own local installation of circom. That's what I do for my project where I already have circom installed in a different path.

**Pull request merged**